autoscheduler/frontend/src/tests/MeetingCard.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import * as React from 'react';
+
+import MeetingCard from '../components/MeetingCard';
+import Meeting, { MeetingType } from '../types/Meeting';
+import Section from '../types/Section';
+import Instructor from '../types/Instructor';
+
+const testSection: Section = {
+  id: 123456,
+  subject: 'CSCE',
+  courseNum: 121,
+  sectionNum: 200,
+  minCredits: 3,
+  maxCredits: null,
+  currentEnrollment: 0,
+  instructor: new Instructor({
+    id: 123456,
+    name: 'Aakash Tyagi',
+  }),
+};
+
+const testMeeting: Meeting = {
+  id: 12345,
+  crn: 123456,
+  building: 'HRBB',
+  meetingDays: new Array(7).fill(true),
+  startTimeHours: 8,
+  startTimeMinutes: 0,
+  endTimeHours: 18,
+  endTimeMinutes: 50,
+  meetingType: MeetingType.LEC,
+  section: testSection,
+};
```

Comment:
Is there a reason `testSection`'s `instructor` uses the constructor, but `testSection` itself and `testMeeting` don't? Honestly I'm not sure if the best practice would be to use them or not, so I'm wondering what your thoughts are

Reply:
Oh. Huh. I didn't even think about that; I just kinda wrote it. I think it would probably be best to convert all of them to using the constructor, since that most closely replicates how the code will actually be used, so I'll fix that, then merge it in.